_data/toc.yaml

```diff
@@ -1885,6 +1885,8 @@ manuals:
       title: Image details view
     - path: /scout/advanced-image-analysis/
       title: Advanced image analysis
+    - path: /scout/web-app/
```

Comment:
**issue:** does this change the path of the docs? If so we'd have to deploy a change in scout.docker.com to reflect that

Reply:
@fdiogo Yes, but I can also add a redirect, depends which you think is quicker / easier?

Reply:
Could we add a redirect and tomorrow we deploy the url change?

Reply:
@fdiogo Done